portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -38,6 +40,22 @@ public DataServlet() {
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
         // Delete everything from previous step.
+        Query query = new Query("Task");
```

Comment:
it looks like this string "Task" is meaningful in that it's a label of some kind for contents of the datastore.

Perhaps this should be factored out and stored as a private static final String in this class. Then it can be used here and on line 66.

Reply:
The same is true for the string "comment-input"